logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -332,10 +332,6 @@ def flush_signal_received?
 
       private
 
-      def iterating?
-        @iterating
-      end
```

Comment:
It is simply a Ruby convention to wrap a boolean in a `<ivar>?` method. It conveys the boolean intent better to other developers.
I wish we could name the ivar itself with a `?` at the end.
Maybe we should adopt Hungarian prefix `b_` for booleans **only** in LS Ruby code? Half joking on here.